src/Containers/WaitTimeDisplay.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled, {useTheme} from "styled-components";
+
+export interface IWaitTimeDisplay {
+
+    /**
+     * The seating/reservation length of average wait time
+     */
+    AverageWaitTime: number,
+}
+
+/**
+ * Primary UI component for user interaction
+ * WaitTimeDisplay
+ */
+export const WaitTimeDisplay: React.FC<IWaitTimeDisplay>
+    = ({
+           AverageWaitTime = 29,
+            ...props
+       }) => {
+
+    return (
+        <Row {...props}>
+            <div>
+                <PieChart AverageWaitTime={AverageWaitTime} />
+            </div>
+
+            <TextFormatting>
+                <div>
+                    <TextAvgWait>Avg. Wait</TextAvgWait>
+                    <TextTime> {AverageWaitTime} Min</TextTime>
+                </div>
+            </TextFormatting>
+        </Row>
+    );
+};
+
+
+/**
+ * This function will determine what color the Pie Chart will be
+ * @param AverageWaitTime {Number} - Number given from getTimeDifference() function
+ * @return {string} - Hexadecimal of color
+ */
+function getColor(AverageWaitTime: Number){
+    const BUSY = 30;
+    const STARTING_TO_GET_BUSY = 15;
+    const {colors} = useTheme();
+
+    if(AverageWaitTime >= BUSY){
```

Comment:
use CSS to change CSS Styling not JavaScript, Move this logic into styled components